src/components/utils/src/push_log.cc

```diff
@@ -100,7 +100,9 @@ void create_log_message_loop_thread() {
   }
 }
 
-void delete_log_message_loop_thread(log4cxx::LoggerPtr& logger) {
+void delete_log_message_loop_thread(const log4cxx::LoggerWeakPtr& del_logger) {
```

Comment:
@SNiukalov Bad parameter name. What does it mean? Delegating_logger? Delegated_logger? Deleting_logger? Deleted_logger? Deleicious_logger? Etc.